crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -68,6 +68,13 @@ type NodeNetworkConfigStatus struct {
 	NetworkContainers []NetworkContainer `json:"networkContainers,omitempty"`
 }
 
+// NetworkContainerStatus defines the status of the Network Containers
+type NetworkContainerStatus struct {
+	Timestamp       metav1.Time `json:"timestamp,omitempty"`
+	LatestStatus    string      `json:"latestStatus,omitempty"`
+	LatestErrorText string      `json:"latestErrorText,omitempty"`
```

Comment:
- Is the "latest" prefix needed? At any given time this struct should already reflect the latest status no?
- Are you only including the error text? What about the numeric error code which is better for handling than arbitrary text?

Reply:
I can get rid of the Latest prefix. The reason I added only the ErrorText was the SubnetFull error was only seen in the Error text and the error code is a more general failure in the DNC code. Had ErrorCode initially but then removed it. Will introduce it back.